### PR TITLE
Fix #1876: Remove the redundant code for fmt.Print

### LINKED ISSUE #1876: Remove the redundant code for fmt.Print
**What would you like to be enhanced**
- Code clean up by removing redundant codes for fmt.Print

**Why is this needed**
- To improve code readability and to provide clear stdout

**Proposed solution**
- Remove the following print codes (or replace it with log.Info().Msg(".....")

https://github.com/cloud-barista/cb-tumblebug/blob/1d56aec43071926557d6cb9fda2f8d11182d3e79/src/core/common/config.go#L152

```
	fmt.Println("UpdateConfig(); ===========================")
	fmt.Println("UpdateConfig(); Key: " + keyValue.Key + "\nValue: " + keyValue.Value)
	fmt.Println("UpdateConfig(); ===========================")
```

---

**Additional note**
- This is for a new contributor!

### PR DESCRIPTION
**Pull request by MohitKambli**
Hello,
Hope you're doing well
As per the request mentioned in issue #1876, I have simple removed the redundant log statement that has been mentioned
If at all I missed something or implemented code changes incorrectly, then do let me know
Sincere apologies for any inconvenience or misunderstanding

Thanks and Regards,
Mohit Kambli

### CODE REVIEW

**Comment by seokho-son** [comment]
Hi @MohitKambli 

Thanks for the contribution! 
Please remove `fmt.Println("UpdateConfig(); Key: " + keyValue.Key + "\nValue: " + keyValue.Value)` as well. We don't need it anymore.

**Comment by MohitKambli** [comment]
Hello @seokho-son,
Will it be fine for you if I raise a new PR altogether for this issue?
Sincere apologies for the inconvenience occurred

Thanks and Regards,
Mohit Kambli

**Comment by seokho-son** [comment]
Hi @MohitKambli 
I guess you've been worried about the build failure in the Checks.

```
Run make
cd src/ && make
make[1]: Entering directory '/home/runner/work/cb-tumblebug/cb-tumblebug/src'
go build -v -o cb-tumblebug
github.com/cloud-barista/cb-tumblebug/src/api/rest/server/model
github.com/cloud-barista/cb-tumblebug/src/core/model
github.com/cloud-barista/cb-tumblebug/src/api/rest/docs
github.com/cloud-barista/cb-tumblebug/src/core/common/label
github.com/cloud-barista/cb-tumblebug/src/core/common
# github.com/cloud-barista/cb-tumblebug/src/core/common
Error: core/common/config.go:1[5](https://github.com/cloud-barista/cb-tumblebug/actions/runs/11481873195/job/31991517015?pr=1878#step:4:6)0:2: declared and not used: keyValue
make[1]: *** [Makefile:2: default] Error 1
make: *** [Makefile:2: default] Error 2
make[1]: Leaving directory '/home/runner/work/cb-tumblebug/cb-tumblebug/src'
Error: Process completed with exit code 2.
```

According to the error messages, 
We need to remove the following code as well. :)
```
keyValue, _ := kvstore.GetKv(key)
```

File: src/core/common/config.go [hunk 1]
```diff
@@ -149,10 +149,6 @@ func UpdateConfig(u *model.ConfigReq) (model.ConfigInfo, error) {
 	}
 	keyValue, _ := kvstore.GetKv(key)
```
Comment (seokho-son) [inline src/core/common/config.go:150]:
```suggestion
```

**Comment by seokho-son** [comment]
You don't have to reopen this PR, we can handle the issue in this PR. :)

**Comment by seokho-son** [comment]
@MohitKambli 
FYI, I applied my suggestion :) So, the check for build test passed.

**Review by seokho-son — APPROVED** [review]
LGTM!

**Comment by seokho-son** [comment]
@all-contributors please add @eltociear for code

**Comment by seokho-son** [comment]
@all-contributors please add @MohitKambli for Code

**Comment by seokho-son** [comment]
@MohitKambli your name is officially listed in https://github.com/cloud-barista/cb-tumblebug?tab=readme-ov-file#contributors- :)

**Pull request merged**